docs/ova/README.md

```diff
@@ -0,0 +1,84 @@
+# Cluster API Provider vSphere, OVA Installer Build process
+
+The build process uses a collection of bash scripts to launch a docker container on your local machine
+where we provision a linux OS, install dependencies, and extract the filesystem to make the OVA.
+
+- [Cluster API Provider vSphere, OVA Installer Build process](#cluster-api-provider-vsphere-ova-installer-build-process)
+  - [Usage](#usage)
+    - [Prerequisites](#prerequisites)
+    - [Build bundle and OVA](#build-bundle-and-ova)
+      - [Build script](#build-script)
+  - [Deploy](#deploy)
+    - [Using the bootstrap cluster](#using-the-bootstrap-cluster)
+
+## Usage
+
+The build process is controlled from a central script, `build.sh`. This script
+launches the build docker container and controls our provisioning and ova
+extraction through `make`.
+
+### Prerequisites
+
+The build machine must have `docker`.
+
+- `docker for Mac`: https://www.docker.com/docker-mac
+- `docker for Windows`: https://www.docker.com/docker-windows
+
+### Build bundle and OVA
+
+#### Build script
+
+The build script pulls the desired versions of each included component into the build container.
+
+> You must specify build step `ova-dev` when calling build.sh
+
+If called without any values, `build.sh` will get Kubernetes version: `stable-1`
+```
+./build/build.sh ova-dev
+```
+
+If called with the values below, `build.sh` will include Kubernetes version v1.12.1
+```
+./build/build.sh ova-dev --kubernetes-version v1.12.1
+```
+
+## Deploy
+
+The OVA must be deployed to a vCenter.
+Deploying to ESX host is not supported at the moment, it might work though.
+
+The recommended method for deploying the OVA:
+- Access the vCenter Web UI, click `vCenter Web Client (Flash)`
+- Right click on the desired cluster or resource pool
+- Click `Deploy OVF Template`
+- Select the URL or file and follow the prompts
+- Power on the deployed VM
+- Open the VM Console and wait for the Kubernetes cluster status to be: RUNNING
+
+Alternative, deploying with `govc` (this also requires [`jq`](https://stedolan.github.io/jq/)):
+
+First create a spec to populate the OVA fields during deployment:
+```console
+$ govc import.spec bin/cluster-api-xxxxxx.ova | jq -r . > cluster-api-spec.json
```

Comment:
Nit-pick: Stylistically, I dislike including the `$` prefix. It's useful in instructions that involve root and non-root steps, but in documents like this I find that it just makes commands harder to copy and paste.

```suggestion
govc import.spec bin/cluster-api-xxxxxx.ova | jq -r . > cluster-api-spec.json
```

(If this change is made, the subsequent code blocks should be updated as well.)

Reply:
Interestingly enough, making it harder to copy/paste is exactly why I like prepending the `$` prompt 😄. 

In general blindly copy/pasting commands from a webpage is a bad practice, this forces the user to at least be aware of what they're doing.

Reply:
Hi @frapposelli,

FWIW, I also include the `$` for this very reason. It makes people consider what they are doing, and prevents blind copy/paste/execute errors.